### PR TITLE
PRO-2473 express

### PR DESCRIPTION
**Pull request by abea**
## Summary

PRO-2473

## What are the specific steps to test this change?

Review for accuracy and clarity

## What kind of change does this PR introduce?
*(Check at least one)*

- [ ] Bug fix
- [ ] New feature
- [ ] Refactor
- [x] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [x] The changelog is updated
- [x] Related documentation has been updated
- [x] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: docs/reference/module-api/module-options.md [hunk 1]
```diff
@@ -126,6 +127,22 @@ module.exports = {
 }
 ```
 
+### `csrfExceptions`
```
Comment (boutell) [inline docs/reference/module-api/module-options.md:130]: That... in other modules... is... a thing. Right. Documentation is good. Thanks for catching this.

File: docs/reference/modules/express.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+## Options
+
+|  Property | Type | Description |
+|---|---|---|
+|`address` | String | Apostrophe listens for connections on all interfaces (`0.0.0.0`) unless this option is set to another address. If the `ADDRESS` environment variable is set, it is used instead. |
+|`apiKeys` | Object | Configure API keys for request authentication. See [the authentication guide](/reference/api/authentication.md#api-keys) for more. |
+|`bodyParser` | Object | The `json` and `urlencoded` properties of this object are merged with Apostrophe's default options to be passed to the [`body-parser` npm module's](https://www.npmjs.com/package/body-parser) `json` and `urlencoded` methods. |
+|`csrf` | Boolean/Object | Set to `false` to disable CSRF protection or to an object with `name` property to customize the CSRF cookie name. See below. |
```
Comment (boutell) [inline docs/reference/modules/express.md:12]: with *a* name property? If we're not being terse ordinarily

File: docs/reference/modules/express.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  // login session timeout.
+  rolling: true,
+  // This option should be customized in every project.
+  secret: 'you should have a secret',
+  name: self.apos.shortName + '.sid',
+  cookie: {}
+}
+```
+
+If you want to use another session store, you can pass an instance, but it's easier to let Apostrophe do the work of setting it up:
```
Comment (boutell) [inline docs/reference/modules/express.md:42]: By default Apostrophe stores sessions in MongoDB so it is not necessary to install another solution. If you want to use another session store, you can pass an instance as the `store` sub-option, but it's easier to let Apostrophe do the work of setting it up:

File: docs/reference/modules/express.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+
+### `csrf`
+
+By default, Apostrophe implements [CSRF protection](https://en.wikipedia.org/wiki/Cross-site_request_forgery) via an `XSRF-TOKEN` cookie. All non-safe HTTP requests (not `GET`, `HEAD`, `OPTIONS` or `TRACE`) automatically receive protection via CSRF middleware, which rejects requests in which the CSRF token does not match the header. If the request was made with a valid API key or bearer token it bypasses this check.
+
+If the `csrf` option is set to `false`, CSRF protection is disabled. **This is not recommended.** Set this option to an object with a `name` property to set that property's value as the CSRF cookie name.
+
+You can configure exceptions to CSRF protection by setting the [`csrfExceptions` option](/reference/module-api/module-options.md#csrfexceptions) of any module to an array of route names specific to that module, or URLs (starting with `/`). Exceptions may use [Minimatch](https://github.com/isaacs/minimatch) wildcards (`*` and `**`).
+
+You may need to use this feature when implementing `POST` form submissions that do not send the header.
```
Comment (boutell) [inline docs/reference/modules/express.md:74]: All requests made with an api key or bearer token are exempt from CSRF protection.

**Review by boutell — CHANGES_REQUESTED** [review]
This is great. Really small stuff.

File: docs/reference/modules/express.md (hunk 4 above)
Comment (abea) [reply docs/reference/modules/express.md:74]: that's in the first paragraph of this section, but I'll emphasize it.

**Review by boutell — APPROVED** [review]

**Pull request merged**